### PR TITLE
Feature/issue 1014 add register form

### PR DESCRIPTION
**Pull request by SteveBarnett**

### CODE REVIEW

File: go/templates/registration/login.html [hunk 1]
```diff
@@ -28,3 +36,26 @@
...
 {% endblock %}
+
+{% block registration_form %}
+
+<h2>Register</h2>
+
+<form action="" method="post">
+    <div class="form-group">
+        <label for="register-name">Your full name
+            <input type="text" id="register-name" name="register-name"class="form-control input-md">
```
Comment (hodgestar) [inline go/templates/registration/login.html]: Missing space before `class`.

**Comment by hodgestar** [comment]
@SteveBarnett -- just checking the current status of this. What's the next step?

**Comment by SteveBarnett** [comment]
@hodgestar Thanks for following up! A grown-up like you or @justinvdm needs to hook up my static registration_form, please.

**Comment by hodgestar** [comment]
Maybe let's try get this in as a 2 hour task for next sprint?

**Comment by SteveBarnett** [comment]
@hodgestar Sounds good to me!

**Comment by badlard** [comment]
:+1:

**Comment by SteveBarnett** [comment]
@badlard Cool, I've updated the text. Now for a grown-up to make it an actual form, not just a static thing. :)

**Comment by hodgestar** [comment]
@badlard @SteveBarnett Did we decide we didn't want this form?

**Comment by SteveBarnett** [comment]
@hodgestar For now, yes. Ben's product positioning document refers.

**Pull request closed without merging**